app/views/flood_risk_engine/enrollments/steps/_local_authority_address.html.erb

```diff
@@ -11,25 +11,35 @@
   <%= t('.select_from_list_label') %>
 </label>
 
-<%# Create prompt manually cos this too flakey to use - e.g disappears on Back
-{ prompt: t('.num_addresses_found', num:  form.address_list.count) } %>
-
-<% form.address_list.unshift({ moniker: t('.num_addresses_found', num:  form.address_list.count) }) %>
-
 <fieldset>
   <%= form_group_and_validation(form, :uprn) do %>
-    <%= f.select :uprn,  options_for_select( form.address_list.collect { |a| [ a[:moniker], a[:uprn] ] }), {},
-                 { include_blank: false,
-                   class: 'form-control form-control-select',
-                   id: 'address_match_selection'
-                 }
+    <%= f.select(
+          :uprn,
+          options_for_select( form.address_list.collect { |a| [ a[:moniker], a[:uprn] ] }),
+          {
+            include_blank: t('.num_addresses_found', num:  form.address_list.count)
```

Comment:
has this been tested in lots of front-end scenarios ? As I mentioned in the comment when I tried the `prompt` suggestion it was possible to get odd behaviour - e.g disappearing prompt - when going Back